tests/migration_test.go

```diff
@@ -1549,17 +1549,30 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				By("tainting the source node as non-schedulabele")
 				tests.Taint(sourceNode.Name, "kubevirt.io/drain", k8sv1.TaintEffectNoSchedule)
 
+				By("waiting until migration kicks in")
+				Eventually(func() int {
+					migrations, err := virtClient.VirtualMachineInstanceMigration(k8sv1.NamespaceAll).List(&metav1.ListOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					runningMigrations := migrations2.FilterRunningMigrations(migrations.Items)
```

Comment:
nit: can you change the import from `migrations2` to `migrations` and the object `migrations` to something else?

Reply:
I have changed the object to `migrationList`, it is descriptive and suggest the content.